kratos/geometries/geometry.h

```diff
@@ -686,6 +686,31 @@ class Geometry : public PointerVector<TPointType>
         return result;
     }
 
+    /** 
```

Comment:
Yes, I added that without asking, and is it not related with the subject of the PR. I will delete it